aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "pagination": {
+    "ListJobs": {
+      "input_token": "Marker",
+      "output_token": "Jobs[-1].JobId",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- e4da5697-70ec-4f8c-8ead-d677942c2b3e -->